src/main/scala/it/unibo/authsim/library/policy/defaults/PolicyDefault.scala

```diff
@@ -1,11 +1,11 @@
-package it.unibo.authsim.library.dsl.policy.defaults
+package it.unibo.authsim.library.policy.defaults
 
-import it.unibo.authsim.library.dsl.Protocol
-import it.unibo.authsim.library.dsl.Protocol.*
-import it.unibo.authsim.library.dsl.cryptography.algorithm.hash.HashFunction.*
-import it.unibo.authsim.library.dsl.policy.builders.PolicyBuilder
-import it.unibo.authsim.library.dsl.policy.defaults.stringpolicy.*
-import it.unibo.authsim.library.dsl.policy.model.Policy
+import it.unibo.authsim.library.Protocol
+import it.unibo.authsim.library.Protocol.*
```

Comment:
Can these 2 imports be merged?

Reply:
IntelliJ throw an error if I do so, you can try by yourself if you want